broker_redis.go

```diff
@@ -66,14 +71,17 @@ type RedisBroker struct {
 }
 
 // DefaultRedisBrokerPrefix is a default value for RedisBrokerConfig.Prefix.
-const DefaultRedisBrokerPrefix = "centrifuge"
+const DefaultRedisBrokerPrefix = "centrifugo"
```

Comment:
I have a question: after the user upgrade, will their messages using the old prefix get lost?

Reply:
Oh, nice catch - that's because I copied this code from Centrifugo where I originally experimented with `rueidis`, should be `centrifuge` of course